app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,13 @@
+<footer>
+  <small>
+    The <a href="http://www.railstutorial.org/">Ruby on Rails Tutorial </a>
```

Comment:
- Dùng gem I18n để sử dụng đa ngôn ngữ, chuyển toàn bộ text thành I18n
- Dùng gem config chuyển "http://www.railstutorial.org/" vào file settings rồi gọi ra ngoài này